src/components/Card/StyledCard.js

```diff
@@ -226,9 +227,9 @@ const StyledCard = ({type, expanded, response}) => {
   )
 }
 StyledCard.propTypes = {
-  type: PropTypes.oneOf(keys(cardTypes)),
+  type: PropTypes.oneOf(['course', 'lesson', 'playlist']),
   response: PropTypes.object,
-  expanded: PropTypes.string
+  expanded: PropTypes.oneOf([false, 'horizontal', 'vertical'])
```

Comment:
Using `oneOf` instead of a generic type is great 👍 Could this array of `[false, 'horizontal', 'vertical']` be determined by the source? Like `keys(someHashWhereTheseThingsAreStored)`